### DISCUSSION TITLE
Doctype in the canvas iframe

### DISCUSSION THREAD

**Discussion by krishnaeverestengineering** [GrapesJS/grapesjs#3521] [Q&A]
Is there a way to add <!DOCTYPE html> as the beginning of the canvas iframe so that the iframe will be in standards mode.

I need this since I am trying to use tinyMce as the custom RTE, and tinyMce requires standards mode.

**Comment by artf** [comment]
It would be better to keep the discussion here https://github.com/artf/grapesjs/issues/3285 as it more like a feature request
